controlsfx/src/main/java/org/controlsfx/validation/ValidationMessage.java

```diff
@@ -84,6 +84,35 @@ public static ValidationMessage error( Control target, String text ) {
     public static ValidationMessage warning( Control target, String text ) {
         return new SimpleValidationMessage(target, text, Severity.WARNING);
     }
+    
+    /**
+     * Factory method to create a simple info message 
+     * @param target message target
+     * @param text message text 
+     * @return info message
+     */
+    public static ValidationMessage info( Control target , String text ) {
+    	return new SimpleValidationMessage(target, text, Severity.INFO);
+    }
+
+    /**
+     * Factory method to create a simple ok message 
+     * @param target message target
+     * @param text message text 
+     * @return ok message
+     */
+    public static ValidationMessage ok( Control target , String text ) {
```

Comment:
The new `INFO` severity is a great addition. Thanks for stepping up for this functionality.

However, I am not able to get around the idea of `OK` severity. Can you please describe as to when this can be useful?

Reply:
I've created a Class named `ControlValidationSupport,` which tracks the validation state of a control and this class has a messageProperty of type `ObjectProperty<ValidationMessage>` and an invalidProperty of type  `BooleanProperty`.

When I have multiple ControlValidationSupport instances for different controls I want to be able to aggregate these messages and show the message with highest severity to the user, for example in the title or heading of a wizard. 
Once all validations are fine I want to display the default or "ok" message and turn the invalid state to false. In my coding I bind the aggregated invalidProperty of _ControlValidationSupport_ instances to the finish button of a wizard.

Does that make sense to you?

Reply:
The OK severity will only be used when all the other validations pass. This can either just remove the previous validation or add its own message. Is that right?

Reply:
Yes that's correct.
I would then consider the ValidationMessage more like a Status class, which represents the overall status of a validation.
In Eclipse RCP wie also have a similar IStatus, which also contains a Severity and a Message.
And it also has OK, INFO, WARNING and ERROR.

Let me know if you still have concerns about this PR.